crates/driver/src/print_build.rs

```diff
@@ -766,8 +777,25 @@ async fn print_file(
     let t = generate_targets(
         opt,
         project_conf,
-        graph_nodes,
-        element,
+        SourceConfig::Main,
+        graph_nodes.clone(),
+        element.clone(),
```

Comment:
if you clone element every time, might as well take a `&String` or `&str` instead IMO.

Reply:
Addressed in ab7edd93aaef9c1b50592e662193c0cc0dfa135c